Doc/library/asyncio-exceptions.rst

```diff
@@ -13,11 +13,12 @@ Exceptions
 
 .. exception:: TimeoutError
 
-   The operation has exceeded the given deadline.
+   A deprecated alias of :exc:`TimeoutError`,
```

Comment:
If you can to deprecated it, you need to add the ".. deprecated::" markup. For now, I suggest to keep it. There is no **need** to remove the alias.

In Python 3.10, we kept the IOError alias, even if it's no longer used.

Reply:
The text was copyed from @tiran PR about replacing a custom `socket.timeout` exception with an alias to `TimeoutError`: https://github.com/python/cpython/pull/23413/files#diff-e9e93d6b76a8a1cf0825d557b12e7ce3e67081cad650063eee520b81fd651943R286

I can change it to something else if you insist.